dcc-portal-ui/app/scripts/common/js/services.js

```diff
@@ -347,4 +347,32 @@
     }
   });
 
+  module.service('EnsureInputService', function(){
```

Comment:
I don't think this should be a service
Would prefer this be a file that exports these functions, and places that use them can import them

I.e.

```js
export function ensureArray = (array) => ...
export function ensureString = ...
export function words = ...
```


```
import {ensureArray, ensureString} from './whateverwerecallingthis.js';
```